pkg/basetypes.go

```diff
@@ -154,3 +154,10 @@ type ReportedUpkeep struct {
 }
 
 type BlockHistory []BlockKey
+
+type LogUpkeepState uint8
+
+const (
+	Eligible LogUpkeepState = iota
+	Performed
+)
```

Comment:
This is for the upkeep state interface correct? I think the goal is to have only final states so let's leave 'Eligible' out.

Reply:
done